dotnet/src/Extensions/Planning.SequentialPlanner/SequentialPlanner.cs

```diff
@@ -39,6 +39,7 @@ public sealed class SequentialPlanner : ISequentialPlanner
         string promptTemplate = prompt ?? EmbeddedResource.Read("skprompt.txt");
 
         this._functionFlowFunction = kernel.CreateSemanticFunction(
+            functionName: this.Config.PlanName?.Replace(" ", ""),
```

Comment:
I think this whitespace replacement should be performed on consumer's side, since it will be hard to cover all cases to handle function name rules:
https://github.com/microsoft/semantic-kernel/blob/f8b6e2af3f862f244ec9c380951a0e1ad5671eb3/dotnet/src/InternalUtilities/src/Diagnostics/Verify.cs#L16

Reply:
I removed the plan name, So now it's not relevant anymore.